.github/workflows/odh-notebooks-sync.yml

```diff
@@ -0,0 +1,114 @@
+# The aim of this GitHub workflow is to update the pipfile to sync with Codeflare-SDK release.
+name: Sync with codeflare-sdk release
+on:
+  workflow_dispatch:
+    inputs:
+      upstream-repository-organization:
+        required: true
+        description: "Provide the upstream repository organization name"
+
+      codeflare-repository-organization:
+        required: true
+        description: "Provide the Codeflare repository organization name"
+
+      codeflare_sdk_release_version:
+        required: true
+        description: "Provide version of the Codeflare-SDK release"
```

Comment:
Is there a reason why it is necessary to gather the upstream/codeflare repos from a user if its just Project CodeFlare and ODH we are syncing to?

Reply:
thanks @Bobbins228 for review, the input params may vary if you need to test same scenario in case of organisation name change or using fork repositories. I have changed these params 
to pass default values if input is not given according to suggestion I received from @sutaakar : 

`--field upstream-repository-organization=${{ github.event.inputs.codeflare-repository-organization || opendatahub-io }}
 --field codeflare-repository-organization=${{ github.event.inputs.codeflare-repository-organization || project-codeflare }}  \`